website/docs/guides/orchestration/webhooks/zapier-ms-teams.md

```diff
@@ -0,0 +1,153 @@
+---
+title: "Post to Microsoft Teams when a dbt Cloud job finishes"
+id: webhooks-guide-zapier-ms-teams
+slug: zapier-ms-teams
+description: Use Zapier and webhooks to send info to Microsoft Teams
+---
+
+This guide assumes familiarity with:
+- dbt Cloud webhooks
+- Zapier
+
+## What does this integration do? 
+When a dbt Cloud job completes:
+ - receive a webhook notification in Zapier
+ - extract the results from the dbt Cloud admin API 
+ - post a summary to a Microsoft Teams channel
+
+![Screenshot of a message in MS Teams showing a summary of a dbt Cloud run which succeeded](/img/guides/orchestration/webhooks/zapier-ms-teams/ms-teams-ui.png)
+
+## Integration steps:
+### Set up the connection between Zapier and Microsoft Teams 
+
+Install the [Zapier app in Microsoft Teams](https://appsource.microsoft.com/en-us/product/office/WA200002044) and [grant Zapier access to your account](https://zapier.com/blog/how-to-automate-microsoft-teams/). 
+
+Note that when you install the Zapier app, it needs to be added to the team whose channel will receive the message.
+
+### Create a new Zap in Zapier
+Use **Webhooks by Zapier** as the Trigger, and **Catch Raw Hook** as the Event. If you don't intend to [validate the authenticity of your webhook](docs/deploy/webhooks#validate-a-webhook) (not recommended!) then you can choose **Catch Hook** instead. 
+
+Press **Continue**, then copy the webhook URL and put it into dbt Cloud. 
```

Comment:
is there a specific place or page (or doc) that specifies where to put the webhook URL?

Reply:
http://localhost:3000/docs/deploy/webhooks#create-a-webhook-subscription would be the closest and we've already linked to that page, but I've reworked this a bit